src/main/java/network/brightspots/rcv/Tabulator.java

```diff
@@ -82,20 +83,27 @@ class Tabulator {
     this.candidateNames = config.getCandidateNames();
     this.config = config;
 
+    fieldIds.initialize(TabulateByField.BATCH);
+    fieldIds.initialize(TabulateByField.PRECINCT);
+
     for (CastVoteRecord cvr : castVoteRecords) {
-      String precinctId = cvr.getPrecinct();
-      if (precinctId != null) {
-        precinctIds.add(precinctId);
+      for (TabulateByField field : config.enabledFields()) {
```

Comment:
good example of the ambiguity here: without context, I would never guess that `config.enabledFields()` is giving me a list of enabled breakdown fields